casper/src/test/scala/coop/rchain/casper/genesis/contracts/TimeoutResultCollectorSpec.scala

```diff
@@ -10,7 +10,7 @@ class TimeoutResultCollectorSpec extends FlatSpec with AppendedClues with Matche
   it should "testFinished should be false if execution hasn't finished within timeout" in {
```

Comment:
This reads awkwardly. Can you change to `"testFinished" should "be false...`?

Reply:
Not related with this PR

Reply:
Loosely related, hence the question. No need to follow the boy scout rule every time